ReplayKitExample/ReplayKitExample/ViewController.swift

```diff
@@ -0,0 +1,286 @@
+//
+//  ViewController.swift
+//  ReplayKitExample
+//
+//  Copyright © 2018 Twilio. All rights reserved.
+//
+
+import UIKit
+import ReplayKit
+import TwilioVideo
+
+class ViewController: UIViewController, RPBroadcastActivityViewControllerDelegate, RPBroadcastControllerDelegate, RPScreenRecorderDelegate, TVIRoomDelegate {
+
+    @IBOutlet weak var spinner: UIActivityIndicatorView!
+    @IBOutlet weak var broadcastButton: UIButton!
+    @IBOutlet weak var conferenceButton: UIButton?
+    @IBOutlet weak var infoLabel: UILabel?
+
+    // Conference state.
+    var screenTrack: TVILocalVideoTrack?
+    var videoSource: ReplayKitVideoSource?
+    var conferenceRoom: TVIRoom?
+
+    // Broadcast state. Our extension will deal with ReplayKit, and connect to a Room.
+    var broadcasting: Bool = false
+    var broadcastController: RPBroadcastController?
+
+    var accessToken: String = "TWILIO_ACCESS_TOKEN"
+    let accessTokenUrl = "http://127.0.0.1:5000/?identity=chris.ios&room=chris"
+
+    static let kStartBroadcastButtonTitle = "Start Broadcast"
+    static let kStopBroadcastButtonTitle = "Stop Broadcast"
+    static let kStartConferenceButtonTitle = "Start Conference"
+    static let kStopConferenceButtonTitle = "Stop Conference"
+    static let kRecordingAvailableInfo = "Ready to share the screen in a Broadcast (extension), or Conference (in-app)."
+    static let kRecordingNotAvailableInfo = "ReplayKit is not available at the moment. Another app might be recording, or AirPlay may be in use."
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        broadcastButton.setTitle(ViewController.kStartBroadcastButtonTitle, for: .normal)
+        conferenceButton?.setTitle(ViewController.kStartConferenceButtonTitle, for: .normal)
+        RPScreenRecorder.shared().delegate = self
+        checkRecordingAvailability()
+
+        // Use RPSystemBroadcastPickerView when available (iOS 12+ devices).
+        #if arch(arm64)
+        if #available(iOS 12.0, *) {
+            // Swap the button for an RPSystemBroadcastPickerView.
+            let broadcastPickerView = RPSystemBroadcastPickerView(frame: CGRect(x: view.center.x-40,
+                                                                                y: view.center.y-40,
+                                                                                width: 80,
+                                                                                height: 80))
+            broadcastPickerView.preferredExtension = "com.twilio.ReplayKitExample.BroadcastVideoExtension"
```

Comment:
Consider reading this from a .plist. We want it to be consistent if the developer chooses a new organization (not com.twilio).